espnet2/tts/tacotron2.py

```diff
@@ -1,10 +1,13 @@
-# Copyright 2018 Nagoya University (Tomoki Hayashi)
+# -*- coding: utf-8 -*-
```

Comment:
`coding: utf-8` has no effects if python3.
pep8 recommends not writing `coding: utf-8` for now.

Reply:
OK! I will remove.